apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -57,9 +62,24 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
     const ExecutorExtra = intl.formatMessage({ id: 'field.Executor.description' })
     const ResponsibleExtra = intl.formatMessage({ id: 'field.Responsible.description' })
 
-    const { action, initialValues, organization } = props
+    const { action: _action, initialValues, organization, afterActionCompleted } = props
     const validations = useTicketValidations()
 
+    const { UploadComponent, syncModifiedFiles: saveFilesToDb } = useMultipleFileUploadHook({ 
```

Comment:
Why you need to rename the method here? Maybe it’s better to rename it inside the hook?

Reply:
Done